packages-common/pluggable-widgets-tools/package.json

```diff
@@ -78,13 +77,14 @@
     "json-loader": "^0.5.7",
     "json-source-map": "^0.6.1",
     "mendix": "^8.7.0",
-    "node-sass": "^4.13.1",
+    "node-fetch": "^2.6.0",
+    "node-sass": "^4.14.1",
     "plugin-error": "^1.0.1",
     "prettier": "^1.19.1",
     "raw-loader": "^3.1.0",
     "react": "~16.9.0",
     "react-dom": "~16.9.0",
-    "react-hot-loader": "^4.12.19",
+    "react-refresh": "^0.8.3",
```

Comment:
- This probably means that the templates introduced in #199 should change to not generate `hot` wrapper.
- What will happen with the existing widgets with that `hot` wrapper? Will they still work?